index.js

```diff
@@ -45,11 +45,15 @@ function openFile (file, sonic) {
   })
 }
 
-function SonicBoom (fd, minLength, sync) {
+function SonicBoom (opts) {
   if (!(this instanceof SonicBoom)) {
-    return new SonicBoom(fd, minLength, sync)
+    return new SonicBoom(opts)
   }
 
+  var { fd, dest, minLength, sync } = opts || {}
+
+  fd = fd || dest
```

Comment:
This looks like it could be `fd = undefined || undefined`. Is that accurate?

Reply:
Yes, this is validated down at line 70. I prefer to keep this double behavior (as it was before).